docs/content/en/docs/configuration/keys.md

```diff
@@ -677,6 +678,7 @@ See also:
 | `allowlist-source-range` | `Path` |         | v0.12 |
 | `denylist-source-range`  | `Path` |         | v0.12 |
 | `whitelist-source-range` | `Path` |         |       |
+| `allowlist-source-header`| `Path` |         | v0.1x |
```

Comment:
This can be merged to v0.13, so please change to v0.13.2 here.

Reply:
done :+1: